File: C++/SegundoParcial/Practico_4/Punto7.cpp
#include <iostream>

using namespace std;

int main(){

    int numero;

    cout << "\nIngrese un numero: ";
    cin >> numero;

    while (numero % 2 == 0)
    {
        cout << "El numero " << numero << " es PRIMO." << endl;
        return 0;
    }

    cout << "El numero " << numero << " NO es primo." << endl;

    return 0;



}

File: Python3/PrimerParcial/Practico 1/Punto7.py
"""
Realizar un programa que solicite al usuario un número entero positivo, y luego en
pantalla muestre solamente los números pares desde el 1 hasta el número
ingresado.
Ej: usuario ingresa 10, en pantalla se mostrará 2 4 6 8 10
"""

numero = int(input("\nIngrese un numero: "))

while numero <= 0:
    print("[!]Numero no valido")
    numero = int(input("\nPorfavor ingrese otro numero: "))

for i in range(2, numero + 1, 2):
    print(i, end= " ")